oci/oci-explanation/ubuntu-oci-container.rst

```diff
@@ -0,0 +1,22 @@
+.. _ubuntu-oci-container-images:
+
+Ubuntu OCI container images
+===========================
+
+The Open Container Initiative (OCI) establishes standards for constructing container 
+images that can be reliably installed across a variety of compliant host environments.
+
+Ubuntu’s `LTS Docker Image Portfolio <https://ubuntu.com/security/docker-images>`_ 
+provides OCI-compliant images that receive stable security updates and predictable 
+software updates, thus ensuring consistency in both maintenance schedule and operational 
+interfaces for the underlying software your software builds on.
+
+Moreover, `Ubuntu-based containers <https://ubuntu.com/containers>`_ (like
+chiselled container images) all leverage the Ubuntu container image as their
```

Comment:
As a newcomer, I don't know what a chiselled container image is. If we have a page that explains them we should link to it here.

Reply:
Y. The Chisel docs will be coming during the next cycle. For now we're documenting it within the Rockcraft docs. I'm adding a reference to it